README.md

```diff
@@ -44,14 +44,15 @@ Preconfigured test are located in `./scale_tests`. Additional scale test should
 #### Adding new requirements
 If new python module is required, added the requirement to `DSS-scalability/requirements.txt`. 
 
-If  using Elasticbeanstock add the new requirements to `DSS-scalability/eb-locustio-sample/.ebextensions/setup.config`
+If using Elastic Beanstalk add the new requirements to `DSS-scalability/eb-locustio-sample/.ebextensions/setup.config`
 under `locust36`.
 
 
 ##### Update chalice requirements
 downloads the packages for aws lambda env
-docker run -it --volume=$PWD:/pp python:3.6 bash -c "pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=/pp" 
+
+    docker run -it --volume=$PWD:/pp python:3.6 bash -c "pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=/pp" 
```

Comment:
that url does not resolve, can you update it?

Reply:
It resolves for me:

```
> pip download invokust locustio==0.8.1 gevent==1.2.2 git+git://github.com/HumanCellAtlas/dcp-cli@tsmith-load-testing#egg=hca --dest=./pp
...
Successfully downloaded invokust locustio gevent hca numpy pyzmq boto3 flask requests msgpack-python six greenlet crcmod cryptography google-auth google-auth-oauthlib Jinja2 jsonpointer jsonschema tweak dcplib argcomplete commonmark docutils PyJWT retrying jmespath s3transfer botocore itsdangerous Werkzeug click certifi chardet idna urllib3 cffi asn1crypto cachetools pyasn1-modules rsa requests-oauthlib MarkupSafe puremagic crc32c future python-dateutil pycparser pyasn1 oauthlib argparse
You are using pip version 19.0.3, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```